src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpStamp.php

```diff
@@ -45,7 +46,12 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
-    public static function createFromAmqpEnvelope(\AMQPEnvelope $amqpEnvelope, self $previousStamp = null): self
+    public function isEnvelopeRedelivered(): bool
```

Comment:
This is really confusing. AMQP also has the concept of redelivery. But this is not the same as the redelivery in the messenger component. Since this is the stamp to encode the amqp flags, this is really unexpected. So this needs a different name or not be exposed at all.

Reply:
This has been renamed